components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -144,6 +144,70 @@ The Chef Infra Server lets you delete environments one at a time. Select **Delet
 
 [Data bags]({{< relref "data_bags" >}}) store global variables as JSON data. Data bags are indexed for searching and can be loaded by a cookbook or accessed during a search.
 
+Chef Infra Server UI lets you:
+
+- Create a data bag.
+- List out all the data bags in one place.
+- Search for a specific data bag item from a list of items.
+- Create a data bag item.
+- Edit a data bag item.
+- Delete a data bag item.
+- Delete a data bag.
+
+#### Create a Data Bag
+
+To create a new data bag, select **Create Data Bag**, as shown below:
+
+{{< figure src="/images/automate/create-data-bag-button.png" alt="Create Data Bag Button">}}
+
+Selecting the **Create Data Bag** button opens a dialog box. Enter the name and select **Create**.
+
+{{< figure src="/images/automate/create-data-bag-popup.png" alt="Create Data Bag Dialog Box">}}
+
+#### Create a Data Bag Item
+
+To create a data bag item, select the data bag created above and follow the steps given below:
+
+Select **Create Item**, as shown below:
```

Comment:
Remove 'as shown below' and describe the location of the button in the step or in the alt-text (don't do both. Alt-text ideally does not repeat content.

```suggestion
Select **Create Item**
```